problems/CountNumSameDigits/src/main/java/Main.java

```diff
@@ -0,0 +1,52 @@
+import java.util.Scanner;
+
+// The task is to count numbers that have all the digits the same from the given range [a, b].
+public class Main {
+
+    public static void main (String[] args){
+
+        String inputRange = "88 99";
+
+        Scanner input = new Scanner(System.in);
+        int noLines = input.nextInt();
+        input.nextLine();
+        int [] arrResults = new int[noLines];
+        for (int i=0; i<noLines; i++){
+
+            String strFromInput = input.nextLine();
+            arrResults[i] = countNumbersWithSameDigits(strFromInput);
+        }
+
+        for (int output:arrResults)
+        { System.out.println(output);
+        }
+    }
+
+    static int countNumbersWithSameDigits(String inputRange){
+        String [] arrRange = inputRange.split(" ");
+        int intStart = Integer.parseInt(arrRange[0]);
+        int intEnd = Integer.parseInt(arrRange[1]);
+
+        int counter =0;
+        for (int i=intStart; i<=intEnd; i++ ){
+            counter += ifNumberWithSameDigits(i);
+        }
+        //System.out.println("counter= "+ counter);
+        return counter;
+    }
+
+    static int ifNumberWithSameDigits(int num){  //returns 1 if number consists of the same digits
```

Comment:
If function name starts with `if` it's expected to return `boolean`. Either rename the function or return boolean